docs/Project-plan.md

```diff
@@ -20,43 +25,80 @@ Based on the requirements of the module, we decided that we are a [Product Team]
 
 Reasons to choose a hybrid style:
-* The team have deliverable deadlines which need to be met to satisfy the customer
-* Addditionally the final deadline can not to be extended, which means we need to be able to be flexible with our commitments when it come closer to the deadline in order to deliver a finsihed product. 
-* We need to maintain a postive work ethic within the team, as without this the efficiency at which tasks are completed will reduce considerably.
-* Helps to understand a common goal for all the memebers, with the understanding that nothing will be achieved if people are not able to comprise. 
 
-**Section explaining how we will deal with people that drop out (illness etc) or not meeting deadlines (sorry not sure what to put)**
+- The team have deliverable deadlines which need to be met to satisfy the customer
+- Additionally the final deadline can not to be extended, which means we need to be able to be flexible with our commitments when it come closer to the deadline in order to deliver a finished product.
+- We need to maintain a positive work ethic within the team, as without this the efficiency at which tasks are completed will reduce considerably.
+- Helps to understand a common goal for all the members, with the understanding that nothing will be achieved if people are not able to compromise.
+
+### Maintaining project flow
 
-In the event that people do not believe the score they get in the peer assessment, members will be allowed to defend their case and suggest why their score should be changed. After which a sutable compromise will be attempted to be reached. However, in the worst case that no compromise is reached we would have to accept the original decision as the orginal score would have been unanimously decided by everyone in the group with certain justification. 
+It is highly likely that the project will be disrupted by unforeseen events, with the most probable scenarios being illness and missed deadlines. All disruptive scenarios will be dealt with on a per-case basis, but as a general rule-of-thumb, we suggest:
+
+- Making a list of jobs the member was working on
+- Re-distributing the cards amongst group members
+- Reflecting on the groups roadmap
+  - Making sure that all future deadlines will be met
+  - If necessary, meeting with the client to explain why a deadline was not met
+  - Resolve conflicts in scheduling
+  - Openly discuss within the group any difficulties in the work
+    - (This is expected - we have a range of technical abilities, it is essential we support each other)
+- Facilitating the member back into the workflow, to avoid loss of contribution and maximise productivity
```

Comment:
> How about this in addition to the first two points:
> 
> * If needed, negotiate deadline extension with the client (The client really doesn't need to know what went wrong especially if it's not a technical problem.)
> * Facilitating the member back into the workflow, to encourage contribution and maximise productivity.

Here I would be transparent with the client all the time and tell them what the cause of the delay is. If we have a problem delivering the product, we should tell the client and also tell them about the solution that we are taking in order to mitigate this problem and to avoid in the future.

Reply:
I agree with George, I think we should both mitigate disruption but also be transparent with the client in delays. I have changed the wording on the facilitating section to be more optimistic.